app/services/sign_in/constants/client_config.rb

```diff
@@ -3,10 +3,13 @@
 module SignIn
   module Constants
     module ClientConfig
-      CLIENT_IDS = %w[mobile web].freeze
-      COOKIE_AUTH = %w[web].freeze
-      API_AUTH = %w[mobile].freeze
-      ANTI_CSRF_ENABLED = %w[web].freeze
+      MOBILE_CLIENT = 'mobile'
```

Comment:
I'm in favor of this PR getting merged as-is to unblock the mobile team, but I'm curious why configuration is half in settings.yml and half in Ruby constants. 

Why not have a more richly structured settings stanza:
```
sign_in_clients:
  - client_id: web
    csrf_enabled: true
    type: cookie
    callback_url: http://localhost:3001/auth/login/callback
  - client_id: mobile
     csrf_enabled: false
     type: api
     callback_url: vamobile://login-success
  - client_id: mobile_test
     ...
```

Reply:
You could argue "you aint gonna need it", but, seems like you already do, and being able to make future changes with just a devops config PR would be nice.

Reply:
You are right, we're starting to hit the limit with the config of the current setup. 
However, the reason it's done this way instead of fully in `settings.yml` is that we're planning on overhauling the whole SiS config setup for a more fully featured, client driven setup, that'll incorporate unified sign in page clients and SiS clients (for example, the idea would be a client like mobile would be able to register `mobile` and `mobile_test` as client_ids, and then be able to update non-prod rules on the fly, through an API or UI. That way mobile would be able to change the redirect_uri as they see fit, or make it a web or app based auth, set max IAL level to 1 or 2, etc).

Anyway, the short of it is I'm trying to keep as much as possible out of settings.yml for now, since we'll be leveraging a file-based or database-based solution for the config, but for now I've still got to keep the different stack settings in there